demos/04_agents/07_agent_routing.py

```
@@ -30,6 +30,16 @@
 
 from demos.shared.utils import can_model_chat, check_model_is_available, get_any_available_chat_model
 
+try:
+    from dotenv import load_dotenv
+except Exception:  # pragma: no cover - optional dependency
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)